INFO: Instrumenting 3608 functions...
Traceback (most recent call last):
  File "target_fuzzer.py", line 102, in <module>
  File "target_fuzzer.py", line 97, in main
NameError: name 'true' is not defined
[145] Failed to execute script 'target_fuzzer' due to unhandled exception!
INFO: Instrumentation complete.
